dashboard/app/controllers/projects_controller.rb

```diff
@@ -696,4 +753,49 @@ def extra_links
       view_options(responsive_content: true)
     end
   end
+
+  # Temporary - will be replaced with storing in database.
+  private def send_project_submission(name, username, project_type, channel_id, description)
+    unless Rails.env.development? || Rails.env.test?
+      subject = 'TESTING: Featured project gallery submission'
+      response = HTTParty.post(
+        'https://codeorg.zendesk.com/api/v2/tickets.json',
+        headers: {"Content-Type" => "application/json", "Accept" => "application/json"},
+        body: {
+          ticket: {
+            requester: {
+              name: username
+            },
+            subject: subject,
+            comment: {
+              body: [
+                "name: #{name}",
+                "user name: #{username}",
+                "project url: https://studio.code.org/projects/#{project_type}/#{channel_id}",
+                "project description: #{description}"
+              ].join("\n")
+            }
+          }
+        }.to_json,
+        basic_auth: {username: 'dev@code.org/token', password: Dashboard::Application.config.zendesk_dev_token}
+      )
+      raise ZendeskError.new(response.code, response.body) unless response.success?
+    end
+  end
+end
+
+# Temporary - will be removed once project submission is stored in database.
+class ZendeskError < StandardError
```

Comment:
^ for example this could also be shared

Reply:
https://github.com/code-dot-org/code-dot-org/pull/61974#discussion_r1819651596